src/charts/helpers/date.js

```diff
@@ -0,0 +1,60 @@
+define(function (require) {
+    'use strict';
+
+    const d3Format = require('d3-format');
+    let idCounter = 0;
+
+    /**
+     * Calculates percentage of value from total
+     * @param  {String}  date    Date
+     * @param  {Number}  days    Number of days to add
+     * @return {String}          Date
+     */
+    const addDays = (date, days) => {
+        const result = new Date(date);
+
+        result.setDate(result.getDate() + days);
+
+        return String(result);
+    }
+
+    /**
+     * Calculate difference between dates in days
```

Comment:
Just to keep comments consistent, `Calculates`.

Reply:
Thanks!